docs/sections/papers/ultrafeedback.md

```diff
@@ -1,3 +1,149 @@
 ## UltraFeedback: Boosting Language Models with High-quality Feedback
 
-...
+UltraFeedback is a large-scale, fine-grained, diverse preference dataset, used for training powerful reward models and critic models.
```

Comment:
perhaps it would be nice to add the image here? or were you trying to keep it lightweight?

Reply:
Yes, ideally this should be lightweight instead, but what image where you thinking of?